SUTrivBot/Repo/triviaQAData.json

```diff
@@ -1,67 +1,1003 @@
+      "answerText": "(Lemon) Jade",
+      "answers": ["Jade", "lemon"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What food did Connie luckily grab before Stevonnie was locked up after the ball?",
+      "answerText": "Protes (protein bar)",
+      "answers": ["Protes", "protein bar"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What's one of the things that helps Steven psychic-project to Earth in Escapsim? ",
+      "answerText": "Beach City, video games, donuts, Dad",
+      "answers": ["Beach City", "video games", "donuts", "Dad"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What two things finishes off Steven's boat in Escapsim?",
+      "answerText": "giant spear, cape",
+      "answers": ["spear", "cape"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "How does Steven defeat the watermelon shark?",
+      "answerText": "by smooching it!",
+      "answers": ["smooching", "kissing"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What does Steven write to Bismuth and Greg in Escapsim?",
+      "answerText": "HELP! WE NEED BACKUP!",
+      "answers": ["HELP", "NEED", "BACKUP"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Where did Pink Diamond get the organic worms from?",
+      "answerText": "Kyanite Colony",
+      "answers": ["Kyanite Colony"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Tell us a Bismuth pun. Can be from the show or original!",
+      "answerText": "don't know what to do with this one ¯\\_(ツ)_/¯ Maybe this will be a mod-only question",
+      "answers": ["¯\\_(ツ)_/¯"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Where does Connie go in the movie?",
+      "answerText": "Space Camp",
+      "answers": ["Space Camp"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "How much of Little Homeworld was complete at the start of the movie?",
+      "answerText": "0.837",
+      "answers": ["0.837"],
```

Comment:
Should change the answer text to "0.837 or 83.7%" and allow either "0.837" or "83.7" as the answer